frontend/workflows/experimentation/src/list-view.tsx

```diff
@@ -10,9 +10,9 @@ import {
   TablePagination,
   TableRow,
   TableSortLabel,
-} from "@material-ui/core";
-import Paper from "@material-ui/core/Paper";
-import { makeStyles } from "@material-ui/core/styles";
+} from "@mui/material";
+import Paper from "@mui/material/Paper";
```

Comment:
Should this be included in the import above from `@mui/material`?

Reply:
Yup, moved.